components/Cases/CaseRecap/CaseNote.tsx

```diff
@@ -13,6 +14,11 @@ interface Props {
 const CaseNote = ({ personId, recordId }: Props): React.ReactElement => {
   const { data: record, error: recordError } = useCase(recordId);
 
+  if (record?.caseFormData?.case_note_description) {
```

Comment:
Do we need to test this?

What is case_note_description is an empty string, malformed JSON, JSON in a format we expect.

Reply:
not happy with this whole bit of the code. we need a more foolproof way to know when to render flexible answers i think